src/steps/person/person-delete.ts

```diff
@@ -15,6 +15,17 @@ export class DeletePersonStep extends BaseStep implements StepInterface {
     description: 'Person\'s email address',
   }];
 
+  protected expectedRecords: ExpectedRecord[] = [{
+    id: 'person',
+    type: RecordDefinition.Type.KEYVALUE,
+    fields: [{
```

Comment:
- [ ] Also recommend adding `created_at` and `updated_at`

Reply:
@iamEAP Thanks for your feedback. We can flatten the `custom_fields` easily for now.